README-Senn.md

```diff
@@ -0,0 +1,72 @@
+# Senn6x
+
+Senn6x is an experimental Residual Channel Attention Network designed for Image Super-Resolution.
+
+<img src="Figures/littleguy.png">
```

Comment:
Usually we try to stay away from HTML code in the Markdown files, can you try this?
`![Little guy](Figures/littleguy.png)`

Reply:
Fixed.